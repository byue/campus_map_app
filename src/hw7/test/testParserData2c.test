# test parser on a graph with 2 nodes triply connected

LoadGraph heroes data2d.tsv
ListChildren heroes a
FindPath heroes a b