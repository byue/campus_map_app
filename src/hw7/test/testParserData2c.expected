# test parser on a graph with 2 nodes triply connected

loaded graph heroes
the children of a in heroes are: b(0.333)
path from a to b:
a to b with weight 0.333
total cost: 0.333